pkg/kyverno/common/common.go

```diff
@@ -0,0 +1,149 @@
+package common
```

Comment:
instead of `common` would this logic be better placed in a package named `files` or `yaml` that describes what it does?

Reply:
I have actually moved common code to `common` package. only `SplitYAMLDocuments` is the function that can go in a package named `yaml`. So I think it's fine for now to have `SplitYAMLDocuments` function in the `common` package. If we get more functions in the future, then we can move to a `yaml` package.

Reply:
ok, sounds good